tomviz/DataSource.h

```diff
@@ -37,9 +37,13 @@ class DataSource : public QObject
   typedef QObject Superclass;
 
 public:
+  /// The type of data in the data source.  The data types currently supported
+  /// are volumetric data and image stacks representing tilt series.
+  enum DataSourceType { VOLUME, TILT_SERIES };
```

Comment:
As this is more Qt/C++ it might be nice to consider using camel-case as these are not C macros. I think we can do this later though, and it doesn't need to be modified in order to get this merged now.